tests/postgres/poi_analysis/use_after_free_test.py

```diff
@@ -12,7 +12,7 @@ def test_poi_use_after_free_read(cpg_db_v2):
         for (free, usage, _, _) in compute_use_after_free(cpg.session, cpg)
     }
 
-    expected = {(20, 24)}
+    expected = {(20, 24), (20, 25)}
```

Comment:
I'm assuming that you checked that this new result is reasonable. If so, everthing LGTM!

Reply:
Yeah, we should have had these as expected results before...